sentry-ruby/lib/sentry-ruby.rb

```diff
@@ -60,6 +63,10 @@ def get_current_hub
       Thread.current[THREAD_LOCAL] || clone_hub_to_current_thread
     end
 
+    def get_transaction
+      get_current_hub.get_transaction
+    end
```

Comment:
I would remove this function, we don't want to have too much surface.
Use `Sentry.get_current_hub.get_scope.get_transaction` instead